installing/installing_oci/installing-c3-assisted-installer.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="installing-c3-assisted-installer"]
+= Installing a cluster on {oci-c3-no-rt} by using the {ai-full}
+:context: installing-c3-assisted-installer
+
+toc::[]
+
+{oci-c3} (C3) lets you run applications and middleware using {oci-first} services on high performance cloud infrastructure in your data center. 
+
+// Overview
+include::modules/c3-assisted-installer-overview.adoc[leveloffset=+1]
+
+// Preparing the Bastian server
+include::modules/c3-assisted-installer-preparing-bastion-server.adoc[leveloffset=+1]
+
+// Terraform script execution - part 1
+include::modules/c3-assisted-installer-running-script-via-home.adoc[leveloffset=+1]
+
+// Preparing the OpenShift image
+[id="c3-assisted-installer-preparing-image_{context}"]
+== Preparing the {oci} image 
+
+Generate the {product-title} ISO image in the {ai-full} on the Red Hat portal. Then convert it to an {oci-first-no-rt} compatible image and upload it to the *Custom Images* page of your {oci-c3-no-rt} environment. 
+
+You can perform this step on your laptop and not on the bastion server or within environments such as Oracle Solution Center. 
```

Comment:
ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#perform
```suggestion
You can generate, convert, and upload the image on your laptop and not on the bastion server or within environments such as Oracle Solution Center. 
```
Please check this revision for accuracy.